pmd-apex/pom.xml

```diff
@@ -13,6 +13,7 @@
 
     <properties>
         <java.version>8</java.version>
+        <kotlin.version>1.7.0</kotlin.version>
```

Comment:
The kotlin version should already be defined in the parent pom

Reply:
Removed